<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="com.hellofresh.challenge.tests.Test_3_Checkout" duration-ms="23677" started-at="2020-01-12T20:55:37Z" finished-at="2020-01-12T20:56:01Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="23677" started-at="2020-01-12T20:55:37Z" finished-at="2020-01-12T20:56:01Z">
      <class name="com.hellofresh.challenge.tests.Test_3_Checkout">
        <test-method status="PASS" signature="setUpTestData()[pri:0, instance:com.hellofresh.challenge.tests.Test_3_Checkout@fdefd3f]" name="setUpTestData" is-config="true" duration-ms="399" started-at="2020-01-12T20:55:37Z" finished-at="2020-01-12T20:55:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestData -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hellofresh.challenge.tests.Test_3_Checkout@fdefd3f]" name="setUp" is-config="true" duration-ms="3632" started-at="2020-01-12T20:55:37Z" finished-at="2020-01-12T20:55:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="validCheckoutTest()[pri:0, instance:com.hellofresh.challenge.tests.Test_3_Checkout@fdefd3f]" name="validCheckoutTest" duration-ms="18557" started-at="2020-01-12T20:55:41Z" description="Valid Checkout  Test" finished-at="2020-01-12T20:55:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.hellofresh.challenge.pages.BasePage.assertTrueContains(BasePage.java:65)
at com.hellofresh.challenge.pages.CheckoutPage.verifyOrderComplete(CheckoutPage.java:67)
at com.hellofresh.challenge.tests.Test_3_Checkout.validCheckoutTest(Test_3_Checkout.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validCheckoutTest -->
        <test-method status="PASS" signature="cleanUp()[pri:0, instance:com.hellofresh.challenge.tests.Test_3_Checkout@fdefd3f]" name="cleanUp" is-config="true" duration-ms="1002" started-at="2020-01-12T20:56:00Z" finished-at="2020-01-12T20:56:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUp -->
      </class> <!-- com.hellofresh.challenge.tests.Test_3_Checkout -->
    </test> <!-- Command line test -->
  </suite> <!-- com.hellofresh.challenge.tests.Test_3_Checkout -->
</testng-results>
